automation-tests/test/scratchOrg.ts

```diff
@@ -0,0 +1,328 @@
+
+    const startDate = Date.now();
+    const durationDays = 1;
+
+    utilities.log(`${this.testSuiteSuffixName} - calling sfdx force:org:create...`);
+    const sfdxForceOrgCreateResult = await exec(`sfdx force:org:create -f ${definitionFile} --setalias ${this.scratchOrgAliasName} --durationdays ${durationDays} --setdefaultusername --json --loglevel fatal`);
+    const json = this.removedEscapedCharacters(sfdxForceOrgCreateResult.stdout);
+    const result = JSON.parse(json).result;
+
+    const endDate = Date.now();
+    const time = endDate - startDate;
+    utilities.log(`Creating ${this.scratchOrgAliasName} took ${time} ticks (${time/1000.0} seconds)`);
+
+    expect(result.authFields).not.toBeUndefined();
+    expect(result.authFields.accessToken).not.toBeUndefined();
+    expect(result.orgId).not.toBeUndefined();
+    expect(result.scratchOrgInfo.SignupEmail).toEqual(EnvironmentSettings.getInstance().devHubUserName);
+
+    // Run SFDX: Set a Default Org
+    utilities.log(`${this.testSuiteSuffixName} - selecting SFDX: Set a Default Org...`);
+    const inputBox = await utilities.executeQuickPick(workbench, 'SFDX: Set a Default Org');
+
+    // Wait for the quick pick list to appear.
+    await utilities.pause(1);
+
+    // Select this.scratchOrgAliasName from the list.
+    let scratchOrgQuickPickItemWasFound = false;
+    const quickPicks = await inputBox.getQuickPicks();
+    for (const quickPick of quickPicks) {
+      const label = await quickPick.getLabel();
+      // Find the org that was created.
+      if (label.includes(this.scratchOrgAliasName)) {
+        await quickPick.select();
+        await utilities.pause(3);
+        scratchOrgQuickPickItemWasFound = true;
+        break;
+      }
+    }
+    expect(scratchOrgQuickPickItemWasFound).toBe(true);
+    // Warning! This only works if the item (the scratch org) is visible.
+    // If there are many scratch orgs, not all of them may be displayed.
+    // If lots of scratch orgs are created and aren't deleted, this can
+    // result in this list growing one not being able to find the org
+    // they are looking for.
+
+    // Look for the success notification.
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Set a Default Org successfully ran');
+    expect(successNotificationWasFound).toBe(true);
+
+    // Look for this.scratchOrgAliasName in the list of status bar items
+    const statusBar = await workbench.getStatusBar();
+    const scratchOrgStatusBarItem = await utilities.getStatusBarItemWhichIncludes(statusBar, this.scratchOrgAliasName);
+    expect(scratchOrgStatusBarItem).not.toBeUndefined();
+
+    utilities.log(`${this.testSuiteSuffixName} - ...finished createDefaultScratchOrg()`);
+    utilities.log('');
+  }
+
+  private get tempProjectName(): string {
+    return 'TempProject-' + this.testSuiteSuffixName;
+  }
+
+  /*
+  private async fixAlias(workbench: Workbench, terminalText: string): Promise<void> {
```

Comment:
Is there a todo that should go with this, or should it be removed?

Reply:
I'll remove it